tools/testbuild.sh

```diff
@@ -239,6 +244,14 @@ function build {
     fail=1
   fi
 
+  # Enable debug symbols if requested. We do this after refresh
```

Comment:
should we move it to configure function?

Reply:
I moved it here otherwise we will try to update the defconf file an CI will fail because there is a change.